assets/css/2-base/_navigation.sass

```diff
@@ -46,13 +46,20 @@
     font-size: 16px
     overflow: hidden
     background: rgba(9,20,154,0.55)
+    transition: all 0.3s ease
+	
+    &:hover
+      background: rgba(9,20,154,0.74)
 
     @media (max-width: $mobile)
       width: 0%
       height: auto
       top: 10rem
       position: fixed
       background: none
+	  
```

Comment:
Changes look great!
I'm just not sure if this is necessary - quick test on the browser looks like it doesn't really make a difference. Otherwise definitely an improvement :)